### PR TITLE
Define tracking id in KYC gatherer

### PR DESCRIPTION
**Pull request by ZengJunyong**
- Add `KycSettingsTracking` for PatchOrganization
- Return `organizationId` from StorefrontGetAccount
- Add a new endpoint `StorefrontGetOrganization`

### CODE REVIEW

File: openapi/components/schemas/Organizations/OrganizationSettings.yaml [hunk 1]
```diff
@@ -19,9 +19,11 @@ properties:
...
   kyc:
     type: object
-    description: The settings for tax calculation.
+    description: The settings for KYC config.
     properties:
       identity-proof:
         $ref: ../KycDocument/Settings/KycSettingsIdentity.yaml
       address-proof:
         $ref: ../KycDocument/Settings/KycSettingsAddress.yaml
+      tracking-id:
```
Comment (subenson) [inline openapi/components/schemas/Organizations/OrganizationSettings.yaml:28]: Suggestion `settings.tracking.document-gatherer.*`.

Comment (ZengJunyong) [reply openapi/components/schemas/Organizations/OrganizationSettings.yaml:28]: done in https://github.com/Rebilly/api-definitions/pull/770/commits/9d4b7c703e23a19ac6485139bc87cb4daec88062

**Review by Boydbueno — DISMISSED** [review]

**Review by subenson — DISMISSED** [review]

File: openapi/components/schemas/Storefront/StorefrontOrganization.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    description: The organization identifier string.
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  name:
+    description: The organization name.
+    type: string
+    maxLength: 60
+  settings:
+    $ref: ../Organizations/OrganizationSettings.yaml
```
Comment (swilczynski) [inline openapi/components/schemas/Storefront/StorefrontOrganization.yaml:15]: We should not expose KYC settings in Storefront endpoints, maybe we could create TrackingOrganizationSettings type and only expose what's needed for tracking?

Comment (ashkarpetin) [reply openapi/components/schemas/Storefront/StorefrontOrganization.yaml:15]: That's correct

Comment (adamaltman) [reply openapi/components/schemas/Storefront/StorefrontOrganization.yaml:15]: I'm not sure this idea solves the underlying problem anyway. I think it needs more consideration.

Comment (ZengJunyong) [reply openapi/components/schemas/Storefront/StorefrontOrganization.yaml]: Anyway, I created `OrganizationTracking` type that only exposes the tracking information. @swilczynski and @ashkarpetin , see this [commit](https://github.com/Rebilly/api-definitions/pull/770/commits/c6726bb8255f478cc1cf314910235898a7282926)

File: openapi/components/schemas/Organizations/OrganizationSettings.yaml [hunk 3]
```diff
@@ -19,9 +19,15 @@ properties:
...
     properties:
       identity-proof:
         $ref: ../KycDocument/Settings/KycSettingsIdentity.yaml
       address-proof:
         $ref: ../KycDocument/Settings/KycSettingsAddress.yaml
+  tracking:
+    type: object
+    description: The tracking config.
+    properties:
+      document-gatherer:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Organizations/OrganizationSettings.yaml:32]: Why is it not camel cased `document-gatherer`?

Comment (ZengJunyong) [reply openapi/components/schemas/Organizations/OrganizationSettings.yaml:32]: I didn't realize this, I just followed the styles like `identity-proof` and `address-proof`

https://github.com/Rebilly/api-definitions/blob/e52b555e17eed3c494751db02f88129acf8bd6fe/openapi/components/schemas/Organizations/OrganizationSettings.yaml#L24

https://github.com/Rebilly/api-definitions/blob/e52b555e17eed3c494751db02f88129acf8bd6fe/openapi/components/schemas/Organizations/OrganizationSettings.yaml#L26

Comment (ZengJunyong) [reply openapi/components/schemas/Organizations/OrganizationSettings.yaml]: I believe `identity-proof` and `address-proof` should be changed to `identityProof` and `addressProof`.
I only updated `document-gatherer` to `documentGatherer` in this PR.

**Pull request closed without merging**